app/views/layouts/application.html.erb

```diff
@@ -26,12 +26,15 @@
           </ul>
           <ul class="nav navbar-nav navbar-right">
             <li>
-                <form class="navbar-form navbar-left" role="search">
-                  <div class="form-group">
-                    <input type="text" class="form-control" placeholder="Search">
-                  </div>
-                  <button type="submit" class="btn btn-default">Submit</button>
-                </form>
+              <%= form_tag '/search', method: 'GET', class: 'navbar-form navbar-left', role: 'search' do %>
+                <%= radio_button_tag :type, 'students', checked: true %>
+                <%= label_tag :type_students, 'students', class: 'radio-label' %>
+                <%= radio_button_tag :type, 'labs' %>
+                <%= label_tag :type_labs, 'labs', class: 'radio-label' %>
+                <%= collection_select :cohort, :id, Cohort.all, :id, :name, {}, { class: 'cohort-select' } %>
+                <%= text_field_tag :query, '', class: 'form-control' %>
+                <%= submit_tag 'Search', class: 'btn btn-default' %>
+              <% end %>
             </li>
           </ul>
       </div>
```

Comment:
refactor this navbar form into a partial. maybe navbar also gets its own partial